float64 size_open
---
bool completed